src/CalendarWeekHeader.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react'
-import { object } from 'prop-types'
+import { object, string } from 'prop-types'
 import { eachDayOfInterval, endOfWeek, startOfWeek, format } from 'date-fns'
 
-export default function CalendarWeekHeader({ locale }) {
+export default function CalendarWeekHeader({ locale, formatWeekday }) {
   const today = new Date()
 
   const weekDays = eachDayOfInterval({
     start: startOfWeek(today, { locale }),
     end: endOfWeek(today, { locale })
-  }).map(date => format(date, 'eee', { locale }))
+  }).map(date => format(date, formatWeekday || 'eee', { locale }))
```

Comment:
Could you instead just pass the prop here and make `'eee'` the default value of the prop (in `CalendarWeekHeader.defaultProps`)?

Reply:
Yes absolutely! Changes coming right up. Thanks for taking a look :)